### PR TITLE
Enable IsPackable for Microsoft.Spark.Extensions.Delta 

### PR DESCRIPTION
**Pull request by AFFogarty**
This is a simple PR that sets `<IsPackable>true</IsPackable>` for the Delta extension.

### CODE REVIEW

File: src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
   </PropertyGroup>
```
Comment (imback82) [inline src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj]: Can you add more fields like you found here: https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark.Experimental/Microsoft.Spark.Experimental.csproj?

Comment (AFFogarty) [reply src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj]: Added the rest of the fields.

**Comment by imback82** [comment]
@AFFogarty can you fix the tests? Now that `<GenerateDocumentationFile>true</GenerateDocumentationFile>` is introduced, it's validating the comments.

**Comment by AFFogarty** [comment]
Made a change to escape the `<` and `>` in Doc code examples.  Otherwise, the build is broken when `<GenerateDocumentationFile>true</GenerateDocumentationFile>` is enabled.

**Pull request merged**